base_product_mass_addition/models/product_product.py

```diff
@@ -0,0 +1,107 @@
+# © 2014 Today Akretion
+# @author Sébastien BEAU <sebastien.beau@akretion.com>
+# @author Mourad EL HADJ MIMOUNE <mourad.elhadj.mimoune@akretion.com>
+# @author Pierrick Brun <pierrick.brun@akretion.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+
+from odoo import api, fields, models
+from odoo.tests.common import Form
+
+
+class ProductProduct(models.Model):
+    _inherit = 'product.product'
+
+    qty_to_process = fields.Float(
+        compute='_compute_process_qty',
+        inverse='_inverse_set_process_qty',
+        help="Set this quantity to create a new line "
+             "for this product or update the existing one."
+    )
+
+    def _prepare_quick_line(self, parent):
+        res = self._get_quick_line_qty_vals()
+        res.update({'product_id': self.id})
+        return res
+
+    def _get_quick_line(self, parent):
```

Comment:
Not sure `quick` is really self explained word ?
- _get_original_line
- _get_parent_line

Reply:
The goal is to make sure there are no clash with another module, _get_parent_line could be used in a lot of contexts